tests/integration/gateway/rateLimit.test.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+jest.setTimeout(2147483647);
+
+const axios = require('axios');
```

Comment:
Same here

Reply:
Need to use axios for rate limit since we need to inspect headers.
helpers/api returns response directly.